src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -120,6 +120,16 @@ public void testDriverVersion() throws SQLException, IOException {
         }
     }
 
+    @Test
+    public void testDatabaseCompatibilityLevel() throws SQLException {
+        try (Connection conn = getConnection()) {
+            SQLServerDatabaseMetaData dbmData = (SQLServerDatabaseMetaData) conn.getMetaData();
+            int compatibilityLevel = dbmData.getDatabaseCompatibilityLevel();
+            assertTrue(compatibilityLevel > 0);
+            assertTrue(compatibilityLevel < dbmData.getDatabaseMajorVersion()*100);
```

Comment:
I don't think we can guarantee this is always going to be true. Also I couldn't find any documentation on correlation between database version and compatibility level I think it's best not to assume this here

Reply:
Here's the link to the MS documentation:
[https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-database-transact-sql-compatibility-level?view=sql-server-ver15](https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-database-transact-sql-compatibility-level?view=sql-server-ver15).
You are correct that in the case of Azure, it is possible that the compatibility level is greater than the database engine version, so I've deleted that check.  It should however always be greater than 0.